uat/testing-features/src/main/resources/greengrass/features/shadow-1.feature

```diff
@@ -395,6 +395,7 @@ Feature: Greengrass V2 ShadowManager
            }
         }
         """
+        Then the local Greengrass deployment is SUCCEEDED on the device after 120 seconds
```

Comment:
how does this prove that shadow works?

You're using `run` now which means that the deployment will be immediately successful, the deployment won't ever fail.

Reply:
This UAT just with deployment assertions is incomplete. I'll have to add assertions to most of these scenarios like their internal UATs.

Reply:
Will do this change for all of the scenarios in the next PR